ember-app/app/models/new/issue.js

```diff
@@ -142,24 +169,18 @@ var Issue = Model.extend({
     }, function(){}, "json");
   },
   stateLabel: function(){
-    var state_label = this.get('other_labels').map((label)=> {
-        return label.name.toLowerCase();
-      }).find((name)=>{
-        return name.toLowerCase() === 'blocked' || name.toLowerCase() === 'ready';
-    });
-    return state_label || "";
+    return this.get('other_labels').map((label)=>{return label.name.toLowerCase()}).find((name)=>{
+      return name === 'blocked' || name === 'ready';
```

Comment:
Oh, I mean passing `this.isStateLabel` as a function as the `filter` callback. `isStateLabel` doesn't use `this` at all, it's a proper function (input :point_right: output).